### ISSUE TITLE
headache should also be a subclass of Abnormality of the head

### ISSUE THREAD

**Issue by yongqunh** [obophenotype/human-phenotype-ontology#5775]
**Headache HP:0002315**

**Suggested revision and reasons**
Headache HP:0002315
https://hpo.jax.org/app/browse/term/HP:0002315
is currently is subclass of Abnormal nervous system physiology
it's true.
meanwhile, I think it should also be a subclass of Abnormality of the head.

**Comment by pnrobinson** [comment]
Abnormality of the head refers to morphological abnormalities. Headache is a physiological abnormality. I think we need to relabel the (old) label to Abnormal head morphology -- I will investigate!

**Comment by yongqunh** [comment]
I see. It makes sense. Thanks.
